packages/headless/src/controllers/commerce/facets/core/headless-core-commerce-facet.ts

```diff
@@ -130,6 +129,18 @@ export type CoreCommerceFacet<
    * @param value - The facet value to evaluate.
    */
   isValueExcluded(value: ValueResponse): boolean;
+  /**
+   * Whether the facet has active values.
+   */
+  hasActiveValues: boolean;
+  /**
+   * Whether the facet can show more values.
+   */
+  canShowMoreValues: boolean;
+  /**
+   * Whether the facet can show less values.
+   */
+  canShowLessValues: boolean;
```

Comment:
Those were originally returned in the state.

However, given that these pieces of information are each derived from the state in some fashion, I think it makes more sense to expose them as getters.

It also makes it somewhat less awkward because the implementation of those getters will differ for category facets.

Reply:
I reverted those changes.